linkedIn.py

```diff
@@ -52,12 +52,14 @@ def search_and_send_request(keywords, till_page, writer):
 
 
 # Login
-driver = webdriver.Chrome(ChromeDriverManager().install())
+driver = webdriver.Chrome('./chromedriver')
```

Comment:
Hello Sandeep, 
Thanks for the work.
Hardcoding anything is never a good idea. Looking forward to more ideas from you.
Regards